.gitlab-ci.yml

```diff
@@ -40,46 +68,123 @@ dev:
     - rake destroy
     - rake clobber
 
-promote-to-stg:
+gcp-dev:
+  stage: dev
+  tags:
+    - gcp
+  script:
+    - cd gcp/live/dev
+    - rake clobber
```

Comment:
Why need to run `clobber` before the rake?

Reply:
Safety. Remember that `after_script` is not guaranteed to complete.

Reply:
I suggest we add `rake destroy_tfstate[k8s]` and `rake destroy_secrets[default]` after `rake clobber`, to forcefully repopulate secrets to make sure that part of secret management code still works as expected.

Reply:
`clobber` is what cleans up all the Volumes, so it wants to go last. How about after `destroy`?

Reply:
Ouch! It won't work after `clobber` in this case, we need to be still authenticated. After `destroy` is the right place.

Reply:
https://github.com/gpii-ops/gpii-infra/pull/94